canonical-web-and-design/charmhub.io
# Add trending graphs to the charms-and-bundles page

## Done

- Add trending graphs to the charms-and-bundles page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045/charms-and-bundles
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See there are some weekly active devices graphs

## Issue / Card

Fixes #234
